pkg/cmd/pulumi-language-java/main.go

```diff
@@ -441,6 +441,12 @@ func (host *javaLanguageHost) GetProgramDependencies(
 	return &pulumirpc.GetProgramDependenciesResponse{}, nil
 }
 
+func (host *javaLanguageHost) RuntimeOptionsPrompts(ctx context.Context,
```

Comment:
If neither are used we can just do `(context.Context, *pulumirpc.RuntimeOptionsRequest)`

Reply:
Formatter ended up complaining at me when I had the named params and told me to use `_`. Since the rest of the file also uses `_` I kept that instead of going to no name at all.